example.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
This is the typical shebang for a python code that is expected to work in python 2 and 3.  You don't give a path to python because it could be anywhere.